server/src/test/java/timely/test/integration/http/HttpApiIT.java

```diff
@@ -80,37 +86,54 @@ public void testMetrics() throws Exception {
         final Server s = new Server(conf);
         s.run();
         try {
-            put("sys.cpu.user " + TEST_TIME + " 1.0 tag1=value1 tag2=value2", "sys.cpu.idle " + (TEST_TIME + 1)
-                    + " 1.0 tag3=value3 tag4=value4", "sys.cpu.idle " + (TEST_TIME + 2)
-                    + " 1.0 tag3=value3 tag4=value4 viz=(a|b|c)", "zzzz 1234567892 1.0 host=localhost");
+            // @formatter:off
+            put("sys.cpu.user " + TEST_TIME + " 1.0 tag1=value1 tag2=value2",
+                "sys.cpu.idle " + (TEST_TIME + 1) + " 1.0 tag3=value3 tag4=value4",
+                "sys.cpu.idle " + (TEST_TIME + 2) + " 1.0 tag3=value3 tag4=value4 viz=(a|b|c)",
+                "zzzz 1234567892 1.0 host=localhost");
+
             // Latency in TestConfiguration is 2s, wait for it
-            sleepUninterruptibly(4, TimeUnit.SECONDS);
+            sleepUninterruptibly(TestConfiguration.WAIT_SECONDS, TimeUnit.SECONDS);
 
             String metrics = "https://localhost:54322/api/metrics";
             // Test prefix matching
             String result = query(metrics);
-            assertTrue(result.contains("<td>sys.cpu.user</td>"));
-            assertTrue(result.contains("<td>tag1=value1 tag2=value2 </td>"));
-            assertTrue(result.contains("<td>sys.cpu.idle</td>"));
-            assertTrue(result.contains("<td>tag3=value3 tag4=value4 </td>"));
-            assertTrue(result.contains("<td>zzzz</td>"));
-            assertTrue(result.contains("<td>host=localhost </td>"));
+            Document doc = Jsoup.parse(result);
+            Elements tableData = doc.select("td");
+
+            assertEquals(1, tableData.select(":contains(sys.cpu.user)").size());
+            assertEquals(1, tableData.select(":contains(tag1=value1 tag2=value2)").size());
+            assertEquals(1, tableData.select(":contains(sys.cpu.idle)").size());
+            assertEquals(1, tableData.select(":contains(tag3=value3 tag4=value4)").size());
+            assertEquals(1, tableData.select(":contains(zzzz)").size());
+            assertEquals(1, tableData.select(":contains(host=localhost)").size());
+
+            //assertTrue(result.contains("<td>sys.cpu.user</td>"));
```

Comment:
just me keeping the original around to look at while I re-wrote the test. I can remove it.